src/assets/styles/scroll.less

```diff
@@ -0,0 +1,40 @@
+.searchScroll {
+  top: 52px;
+  bottom: 0px;
+  position: fixed;
+  padding-bottom: 30px;
+  padding-bottom: 0px;
+}
+
+.searchScroll .mCSB_scrollTools {
+  left: 2px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.favScroll {
+  top: 52px;
+  bottom: 0px;
```

Comment:
If jQuery needs two classes that's okay, but the css can still be tidied, e.g. use `.favScroll, .searchScroll` in the two duplicated places.